android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/MainActivity.java

```diff
@@ -375,11 +385,80 @@ public void onConfigurationChanged(Configuration newConfig) {
     invalidateOptionsMenu();
   }
 
+  @Override
+  public boolean onPrepareOptionsMenu(final Menu menu) {
+    final MenuItem _overflowMenuItem = menu.findItem(R.id.action_overflow);
+    if(_overflowMenuItem==null)
```

Comment:
This fixed the NPE in landscape. But now I don't see the new menu item in portrait or landscape

![menu](https://user-images.githubusercontent.com/7358010/69059055-5f892280-0a47-11ea-8aed-dd6a130ae08e.png)

Reply:
ah, it's cause the update count was 0. I think it's preferable to still display it, but be disabled/grayed out like we do in other menus

Reply:
I'm actually comfortable with not displaying the update menu item at all when there are no updates. It behaves more like a 'banner' that attracts attention that way.

Reply:
Landscape is working now